doc/install/build-ceph.rst

```diff
@@ -32,10 +32,9 @@ repository and execute the following::
     cd build
     make
 
-.. note:: By default do_cmake.sh will build a debug version of ceph that may
-   perform up to 5 times slower with certain workloads. Pass 
-   '-DCMAKE_BUILD_TYPE=RelWithDebInfo' to do_cmake.sh if you would like to
-   build a release version of the ceph executables instead.
+.. note:: By default do_cmake.sh will build a release version of ceph with debug
+   symbols. Pass '-DCMAKE_BUILD_TYPE=Debug' to do_cmake.sh if you would like to
+   build a debug version of the ceph executables instead.
```

Comment:
There;s a similar passage in README.md

Reply:
thanks, the document changes are dropped. as #30799 addresses the backward compatibility issue.